src/debuginfo.cpp

```diff
@@ -343,7 +343,9 @@ class JuliaJITEventListener: public JITEventListener
         for (const object::SymbolRef &sym_iter : debugObj.symbols()) {
             StringRef sName;
 #ifdef LLVM37
-            sName = sym_iter.getName().get();
+            auto sNameOrError = sym_iter.getName();
+            assert(sNameOrError);
```

Comment:
Is the issue that `bool(sNameOrError)` has to be called? It seems that this won't work for release build where assertion is disabled by default.